docs/deep/djl.md

```diff
@@ -233,7 +233,7 @@ artifacts = artifacts.findAll(a -> a.properties.getOrDefault('artist', null))
 // Note: this shouldn't be too big! Define a maximum dimension
 double maxDim = 1024
 def server = getCurrentServer()
-double downsample = Math.max(server.getWidth(), server.getHeight()) / maxDim
```

Comment:
For some reason a division here breaks the code highlighting. This is the lazy option rather than debugging why

Reply:
Yeah, noticed it broke the highlighting but never found a fix for it.

Yours is clever - but is it maybe *too* clever...? I tend towards accepting the broken highlighting but not sure.

Reply:
Might be, I'd suggest changing the comment before to "scale image dimensions by the inverse of the maximum dimension we want to allow" which maybe makes it clearer

Reply:
I think `/ maxDim` remains strongly preferable for clarity (even if it messes the syntax highlighting).

Could you change it back please so the PR can be merged? Would edit it myself, but when I started I got the sense I was going to make a git-mess.

Reply:
How do you feel about switching from groovy to Java syntax highlighting, and going back to division? Does mean changing from def to var but I don't think the difference is meaningful here

![Screenshot from 2023-09-12 13-27-18](https://github.com/qupath/qupath-docs/assets/10779688/766d4847-3f53-4bc8-9945-7b238ada20ff)

Reply:
> but it's highlighted as a symbol rather than a keyword

Using Java highlighting with `var` sounds ok, but we should check when updating docs for v0.5.0.

The main thing is really to minimize confusion. Switching to `var` is inconsistent with most Groovy docs, but [looks like it should do the same thing](https://groovy-lang.org/semantics.html#_variable_definition).

I already tried introducing `var` with the StarDist docs, but then abandoned it for some reason. I recall `def` and `var` were almost interchangeable... but not quite. But I can't remember the time they differed, so really we just need to look out for it.

Reply:
I think you can't define functions with `var`, at least

Reply:
Ah, I don't think that's what I encountered - but is a far more relevant reason not to switch fully to Java. 

Let me know when it's done, or go ahead and merge. The [build preview](https://qupath--75.org.readthedocs.build/en/75/docs/deep/djl.html) isn't showing any syntax highlighting at all now for the *style transfer* section.

Reply:
Ah yeah, Java hates single quotes. Should be fine to merge once the build preview looks ok for that section